DemoViperUITestsKIF/Specs/ListReposUISpecs.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ListReposSpecs.swift
+//  DemoViperUITestsKIF
+//
+//  Created by Nguyen M. Tam on 07/01/2021.
+//  Copyright © 2021 Nimble Co. Ltd. All rights reserved.
+//
+
+import Quick
+import Nimble
+import KIF
+
+@testable import DemoViper
+
+final class ListReposUITests: QuickSpec {
+
+    override func spec() {
+
+        let tester = self.tester()
+
+        describe("list repos") {
+
+            beforeEach {
+                setupTestEnvironment()
+                self.continueAfterFailure = true
+            }
+
+            it("open the list repo screen successfully") {
+               let view = tester.waitForView(withAccessibilityLabel: ListRepos.view.identifier)
+                expect(view).notTo(beNil())
+            }
+
+            context("no repo is added to bookmark") {
+
+                it("verify initial data of a repo and open the detail repo screen") {
+                    let repoCellItem = tester.tapRepoCell(at: 0)
+                    expect(repoCellItem?.repoFullName) != ""
```

Comment:
What do you think about adding `isEmpty` property into `Repo` model? 
I saw we have a duplicate code at line 52